recipe/meta.yaml

```diff
@@ -10,13 +10,18 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    # github.com/sodre/yq:entry_points: replace scripts with entry-points
+    - github_com__sodre__yq__entry_points.patch
 
 build:
-  skip: True  # [win]
-  number: 0
+  number: 1
+  noarch: python
```

Comment:
Using entry-points allows us to build a noarch: python package. However, keep in mind that Windows builds will still fail to install because of conda-forge/jq-feedstock#3.